common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -115,13 +117,24 @@ public class WebViewAuthorizationFragment extends AuthorizationFragment {
 
     private PermissionRequest mCameraPermissionRequest;
 
+    private ActivityResultLauncher<LegacyFido2ApiObject> mFidoLauncher;
+
     @Override
     public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        final String methodTag = TAG + ":onCreate";
         final FragmentActivity activity = getActivity();
         if (activity != null) {
             WebViewUtil.setDataDirectorySuffix(activity.getApplicationContext());
         }
+        if (CommonFlightsManager.INSTANCE.getFlightsProvider().isFlightEnabled(CommonFlight.ENABLE_LEGACY_FIDO_SECURITY_KEY_LOGIC)) {
+            mFidoLauncher = registerForActivityResult(
```

Comment:
mFidoLauncher is technically needed only in <= 13, right?

Reply:
True; I anticipate I will need to add in some null checks, so will implement this tomorrow

Reply:
086326adf883f7e09057ea2de33b10caf9745529